templates/taxi/car_list.html

```diff
@@ -7,6 +8,13 @@ <h1>
       Create
     </a>
   </h1>
+
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" class="btn btn-primary">
+  </form>
```

Comment:
Move the search form to a separate template and include this template where you need a search form